src/asgen/engine.d

```diff
@@ -320,10 +320,13 @@ public:
 
         // prepare icon-tarball array
         immutable iconTarSizes = ["64", "128"];
+        immutable iconTarScales = ["1", "2"];
```

Comment:
I wonder whether we should iterate over `wantedIconSizes` here, instead of defining another array (minor thing though).